Large rework

YKraml opened the pull request:
Redone the complete implementation.

coemgen1992 commented:
There are some inconsistencies, which I found. Please change them, so that the request can be merged.

1. pom.xml
- placeholder for the local jar of travart-core should be replaced witht the actual group and artifact id
<groupId>at.jku.cps</groupId>
<artifactId>travart-core</artifactId>
<version>2.0.0</version>

2. Version in Plugin impl is differnt as the one in the pom.xml -> update to 2.0

3. FeatureAndGroupHandlerImpl: Visibility of the question Strings is inconsistent for no apparent reason. Please make the consistently the same visibility or comment why they are not private if made not private.

4. package definition: travart missing
package definition should edu.kit.travart.dopler.* by travart convention

YKraml commented:
So, I have implemented all the suggestions.
In addition, the README is almost done.

YKraml commented:
So, the sampling package is now also implemented. But only works for small decision models.

YKraml commented:
Implemented the stress tests. About 2000 test cases that run through. They are taken from https://github.com/SoftVarE-Group/feature-model-benchmark.

I have removed the 150 largest UVL models in the data set. My computer could not handle the load in a reasonable time, and I would rather not balloon the size of the repo by 2 GB for only these few test cases.

YKraml commented:
Added the package docs
